tests/std/tests/P0896R4_views_common/test.cpp

```diff
@@ -387,8 +387,128 @@ struct instantiator {
     }
 };
 
+template <class T>
+struct difference_type_only_iterator {
+    static_assert(is_object_v<T>);
+
+    friend constexpr bool operator==(difference_type_only_iterator, difference_type_only_iterator)  = default;
+    friend constexpr auto operator<=>(difference_type_only_iterator, difference_type_only_iterator) = default;
+
+    using iterator_concept = contiguous_iterator_tag;
+    using value_type       = remove_cvref_t<T>;

```

Comment:
No change requested: After `static_assert(is_object_v<T>)` which forbids references, it would be sufficient to say `remove_cv_t<T>`.